lib/spack/spack/cmd/style.py

```diff
@@ -48,6 +48,13 @@ def grouper(iterable, n, fillvalue=None):
 #: tools we run in spack style
 tools = {}
 
+#: warnings to ignore in mypy
+mypy_ignores = [
+    # same as `disable_error_code = "annotation-unchecked"` in pyproject.toml, which
+    # doesn't exist in mypy 0.971 for Python 3.6
```

Comment:
Mentioned this in chat, but we prob don't need to support running mypy with Python 3.6.

Reply:
Going to fix this and other tools in another PR